iron-a11y-keys-behavior.html

```diff
@@ -321,14 +286,12 @@
 
       keyboardEventMatchesKeys: function(event, eventString) {
         var keyCombos = parseEventString(eventString);
-        var index;
-
-        for (index = 0; index < keyCombos.length; ++index) {
-          if (keyComboMatchesEvent(keyCombos[index], event)) {
+        var eventKey = normalizedKeyForEvent(event);
+        for (var i = 0, l = keyCombos.length; i < l; ++i) {
```

Comment:
While this optimization is well-reasoned, modern JavaScript engines are pretty good at optimizing this without the need to create a second temporary variable to hold the value of length.

http://jsperf.com/for-loop-length-memoization